ml/src/main/scala/org/apache/spark/ml/h2o/models/H2OMOJOModel.scala

```diff
@@ -46,7 +46,7 @@ class H2OMOJOModel(val mojoData: Array[Byte], override val uid: String)
   // Some MojoModels are not serializable ( DeepLearning ), so we are reusing the mojoData to keep information about mojo model
   @transient var easyPredictModelWrapper: EasyPredictModelWrapper = _
 
-  case class BinomialPrediction(p0: Double, p1: Double)
+  case class BinomialPrediction(p0: Double, p1: Double, p0_calibrated: Double, p1_calibrated: Double)
```

Comment:
Apologize for the confusion with ```NaN``` vs. ```null```. ```Double``` is a value type is in Scala.
```
scala> val d = null.asInstanceOf[Double]
d: Double = 0.0
```
So I'm wondering how we gonna specify ```p0_calibrated```, ```p1_calibrated``` when are not defined... ```Option```, ```java.lang.Double```, ```Double.NaN```?